creates.sql

```diff
@@ -0,0 +1,18 @@
+CREATE DATABASE movies;
+
+USE movies;
+
+CREATE TABLE movies(
+id INT AUTO_INCREMENT,
+title VARCHAR(225),
+directors VARCHAR(225), 
```

Comment:
Directors should be its own table since there is a many-to-many relationship between directors and movies.